### PR TITLE
Consolidate private constants

### LINKED ISSUE #29: Move private constants to SDLPrivateSecurityConstants
For consistency, the `availableMakes` property should be moved to the `SDLPrivateSecurityConstants` class. 

The README should be updated to reflect this change.

### PR DESCRIPTION
**Pull request by NicoleYarroch**
Fixes #29 

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Testing Plan
* Smoke tests
* Added test cases

### Summary
* Moved setting of `availableMakes` to `SDLPrivateSecurityConstants`
* Renamed two private classes to remove underscore:  `SDLTLSEngine` and `SDLCertificateManager`

##### Enhancements
* Consolidate setting of private constants in `SDLPrivateSecurityConstants`.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: SDLSecurity/SDLPrivateSecurityConstants.h [hunk 1]
```diff
@@ -8,11 +8,14 @@
 
 #import <Foundation/Foundation.h>
 
-
+extern NSSet<NSString *> *availableMakes;
```
Comment (joeljfischer) [inline SDLSecurity/SDLPrivateSecurityConstants.h]: This should be a class property on the below interface

File: SDLSecurity/SDLSecurityManager.m [hunk 2]
```diff
@@ -88,7 +89,7 @@ - (nullable NSData *)decryptData:(NSData *)data withError:(NSError * _Nullable _
 }
 
 + (NSSet<NSString *> *)availableMakes {
-    return [NSSet setWithArray:@[@"Example"]];
+    return [SDLPrivateSecurityConstants.class availableMakes];
```
Comment (joeljfischer) [inline SDLSecurity/SDLSecurityManager.m]:
```suggestion
    return SDLPrivateSecurityConstants.availableMakes;
```

Once you make the above change, this should be possible.

**Review by joeljfischer — CHANGES_REQUESTED** [review]

File: SDLSecurity/SDLPrivateSecurityConstants.h (hunk 1 above)
Comment (NicoleYarroch) [reply SDLSecurity/SDLPrivateSecurityConstants.h]: Fixed.

File: SDLSecurity/SDLSecurityManager.m (hunk 2 above)
Comment (NicoleYarroch) [reply SDLSecurity/SDLSecurityManager.m]: Fixed.

**Review by joeljfischer — APPROVED** [review]

**Pull request merged**